src/App.tsx

```diff
@@ -8,7 +8,7 @@ function App() {
   return (
     <div className="App">
       <DndProvider backend={HTML5Backend}>
-        <CardDeck cardsURL="card-deck.json" />
+        <CardDeck cardsURL="http://localhost:3000/api/cards" />
```

Comment:
you have added `proxy` in package.json, therefore you should be able to omit domain part of the url
```
- <CardDeck cardsURL="http://localhost:3000/api/cards" />
+ <CardDeck cardsURL="/api/cards" />
```
docs: `proxy tell the app to proxy any unknown requests to your API server`
https://create-react-app.dev/docs/proxying-api-requests-in-development/